websites/K/KaniWork/dist/metadata.json

```diff
@@ -4,21 +4,22 @@
     "name": "fantomitechno 🦊",
     "id": "563749920683720709"
   },
-  "url": "kanishiel.com",
+  "url": "kaniwork.com",
   "description": {
-    "en": "KaniShiel is a discord bot that can host multiple games on an indefinite number of simultaneous servers.",
-    "fr": "KaniShiel est un bot de jeu qui peut animer de multiple jeu sur un nombre indéfini de serveur simultané."
+    "en": "The Kani Network is a community gathered around different discord bots",
+    "fr": "Le Kani Network est une communauté réunis autour de différents bots discord"
   },
-  "service": "KaniShiel",
-  "version": "1.2.1",
+  "service": "KaniNetwork",
+  "version": "1.3.0",
```

Comment:
service name and folder name must be the same

Reply:
Oops, forgot about that I modify this